lib/oppen.rb

```diff
@@ -94,72 +102,74 @@ def trim_trailing_whitespaces? = @trim_trailing_whitespaces
-      new(indent_anchor: IndentAnchor::ON_BEGIN, eager_print:, trim_trailing_whitespaces:, upsize_stack:)
+      new(eager_print:, indent_anchor: IndentAnchor::ON_BEGIN, trim_trailing_whitespaces:, upsize_stack:)
     end
   end
 
   # @param value [String]
-  # @param width [Integer] token width that defaults to value.length
+  # @param width [Integer] token width that defaults to value.length.
   #
-  # @return [Oppen::Token::String] a new String token
+  # @return [Token::String] a new String token.
   def self.string(value, width: value.length)
     Token::String.new(value, width:)
   end
 
   # @see Token::Whitespace
   #
-  # @return [Oppen::Token::Whitespace] a new Whitespace token.
+  # @return [Token::Whitespace] a new Whitespace token.
   def self.whitespace(value)
     Token::Whitespace.new(value, width: value.bytesize)
   end
 
-  # @param str [String]
-  # @param line_continuation [String] If a new line is needed display this string before the new line
-  # @param offset [Integer]
-  # @param width [Integer] token width that defaults to str.length
+  # @param str               [String]
+  # @param line_continuation [String]  If a new line is needed display this string before the new line.
+  # @param offset            [Integer]
+  # @param width             [Integer] Token width that defaults to str.length.
   #
-  # @return [Oppen::Token::Break] a new Break token
+  # @return [Token::Break] a new Break token.
   def self.break(str = ' ', line_continuation: '', offset: 0, width: str.length)
     Token::Break.new(str, width:, line_continuation:, offset:)
   end
 
-  # @param line_continuation [String] If a new line is needed display this string before the new line
-  # @param offset [Integer]
+  # @param line_continuation [String] If a new line is needed display this string before the new line.
+  # @param offset            [Integer]
   #
-  # @return [Oppen::Token::LineBreak] a new LineBreak token
+  # @return [Token::LineBreak] a new LineBreak token.
   def self.line_break(line_continuation: '', offset: 0)
     Token::LineBreak.new(line_continuation:, offset:)
   end
 
   # @param offset [Integer]
   #
-  # @return [Oppen::Token::Begin] a new consistent Begin token
+  # @return [Token::Begin] a new consistent Begin token.
   def self.begin_consistent(offset: 2)
     Token::Begin.new(break_type: Token::BreakType::CONSISTENT, offset:)
   end
 
   # @param offset [Integer]
   #
-  # @return [Oppen::Token::Begin] a new inconsistent Begin token
+  # @return [Token::Begin] a new inconsistent Begin token.
   def self.begin_inconsistent(offset: 2)
     Token::Begin.new(break_type: Token::BreakType::INCONSISTENT, offset:)
   end
 
-  # @return [Oppen::Token::End] a new End token
+  # @return [Token::End] a new End token.
   def self.end
     Token::End.new
   end
 
-  # @return [Oppen::Token::EOF] a new EOF token
+  # @return [Token::EOF] a new EOF token.
```

Comment:
Description. What do they do. `eof` for instance have a very particular effect, no? All tokens after it are ignored …

Reply:
`eof` can be considered as a `flush`, we can have multiple ones in the tokens list (even thought it is useless since the only logical position of it is at the end of the list).